bisqapps/shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/uicases/offers/OffersListScreen.kt

```diff
@@ -0,0 +1,88 @@
+package network.bisq.mobile.presentation.ui.uicases.offers
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.blur
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import network.bisq.mobile.presentation.ViewPresenter
+import network.bisq.mobile.presentation.ui.components.layout.BisqStaticScaffold
+import network.bisq.mobile.presentation.ui.components.molecules.*
+import org.koin.compose.koinInject
+
+interface IOffersList : ViewPresenter {
+    fun takeOffer()
+}
+
+@Composable
+fun OffersListScreen() {
+    val presenter: ICurrencyList = koinInject()
+    val states = listOf(
+        "Buy from",
+        "Sell to"
+    )
+    val openDialog = remember { mutableStateOf(false) }
+    val rootNavController: NavController
+
+    LaunchedEffect(Unit) {
+        presenter.onViewAttached()
+    }
+
+    BisqStaticScaffold(
+        topBar = {
+            TopBar(title = "Offers")
+        },
+    ) {
+        Box(modifier = Modifier.fillMaxSize().blur(if (openDialog.value) 12.dp else 0.dp)) {
+            Column {
+                Column(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalAlignment = Alignment.CenterHorizontally
+                ) {
+                    StateToggle(states, 130.dp)
+
+                    Spacer(modifier = Modifier.height(32.dp))
+                    LazyColumn(
+                        modifier = Modifier.padding(10.dp),
+                        verticalArrangement = Arrangement.spacedBy(18.dp)
+                    ) {
+                        items(3) {
+                            OfferCard(onClick = {
+                                openDialog.value = !openDialog.value
+                            })
+                        }
+                    }
+
+                    if (openDialog.value) {
+                        BisqDialog {
+                            ConfirmationDialog(
+                                message = "Do you want to take this trade?",
```

Comment:
Are we dropping the dependency on the translator or we forgot to add it here? 

I'm ok if we hardcode in english for the MVP just want to make sure is a conscious decision.

Reply:
Do we need the dialogue? In Bisq Easy we dont have it but open the take offer screens (select the available options or get review screen if nothing to select). Though we have plenty of warnings with the reputation based amount limits. That will be painful to replicate ;-(.

Reply:
great question. I think we should just go to the trade details and the user can decide if they want to take it there?
For now we can just log something on click to indicate the todo

good that he developed a dialog though, I'm sure it will be handy somewhere :)  ("Confirm Trade paid? Y/N e.g.")

Reply:
> Are we dropping the dependency on the translator or we forgot to add it here?
> 
> I'm ok if we hardcode in english for the MVP just want to make sure is a conscious decision.

My bad!! Missed it. I will add the translation strings everywhere

Reply:
Yes. Will remove the Dialog from here